### PR TITLE
support for ubuntu 16

### PR DESCRIPTION
**Pull request by grantjoy**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,5 +1,8 @@
 FROM golang:1.6
 
+RUN apt-get update
```
Comment (gsquire) [inline Dockerfile:3]: We need to combine this statement with the install per [Maynard's investigation](https://distil.slack.com/archives/GDVS9S2PR/p1552609394040200).

File: debian/Dockerfile.debuild [hunk 2]
```diff
@@ -0,0 +1,11 @@
+FROM golang:1.6
```
Comment (gsquire) [inline debian/Dockerfile.debuild]: I thought go.rice needed go >= 1.8?

File: debian/docker_package.sh [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+# Make the directory with the assets will care about.
+SHORT_HASH=$(git rev-parse --short HEAD)
+VERSION="1~$SHORT_HASH-1"
+
+# This should be unique to your repository.
+DIR="cfssl-$VERSION"
+
+# Insert the version into the changelog.
+sed -i "s/VERSION/$VERSION/g" debian/changelog
```
Comment (gsquire) [inline debian/docker_package.sh]: We need to add an extension to the `-i` argument or else it won't work locally. You can them remove the backup files after running `sed`.

File: debian/rules [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+# As a special exception, when this file is copied by dh-make into a
+# dh-make output file, you may use that output file without restriction.
+# This special exception was added by Craig Small in version 0.37 of dh-make.
+
+# Uncomment this to turn on verbose mode.
+export DH_VERBOSE=1
+%:
+	dh $@
+override_dh_auto_install:
+	mkdir -p debian/tmp
```
Comment (gsquire) [inline debian/rules]: This isn't necessary with the `mkdir` below.

File: debian/rules [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+	mkdir -p debian/tmp/src
+	cp -r /var/lib/jenkins/jobs/cfssl/jobs/package/workspace/cfssl/*  debian/tmp/src
```
Comment (gsquire) [inline debian/rules]: Use a relative path?

**Review by gsquire — CHANGES_REQUESTED** [review]
Did you want to include these binaries in git? Maybe add them to `.gitignore`?

File: debian/rules (hunk 5 above)
Comment (grantjoy) [reply debian/rules]: changed, thanks.

**Comment by grantjoy** [comment]
@gsquire thanks for the feedback. I made a lot of changes, mainly removing docker.

**Review by gsquire — COMMENTED** [review]
Can we remove the docker related files if they are unused then?

**Comment by grantjoy** [comment]
@gsquire for now I just want to leave the binaries in. at some point we could refactor them out.

**Review by gsquire — APPROVED** [review]
LGTM

**Pull request merged**